src/cr/cube/dimension.py

```diff
@@ -479,6 +487,14 @@ def shape(self):
         """int count of *all* elements in this dimension, both valid and missing."""
         return len(self.all_elements)
 
+    @lazyproperty
+    def subtotal_labels(self):
+        """tuple of string element-labels for each valid subtotal in this dimension.
+
+        Element-labels appear in the order defined in the cube-result.
+        """
+        return tuple(e.label for e in self.subtotals)
```

Comment:
I think `s` instead of `e`, wouldn't you? like `s.label for s in self.subtotals`?